test-data/unit/fixtures/dict.pyi

```diff
@@ -30,6 +31,7 @@ class dict(Mapping[KT, VT]):
     @overload
     def get(self, k: KT, default: Union[KT, T]) -> Union[VT, T]: pass
     def __len__(self) -> int: ...
+    def keys(self) -> dict_keys[KT, VT]: ...
```

Comment:
This was just the easiest way to get things working, happy to hear alternatives if there's some other approach I'm not aware of